Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -584,6 +584,12 @@ def audit_homepage
       problem "Please use https:// for #{homepage}"
     end
 
+    case homepage
+    when %r{^http://([^/]*)sourceforge\.net(/|$)}i,
+         %r{^http://([^/]*)sf\.net(/|$)}i
+      problem "#{homepage} should be `https://#{$1}sourceforge.io/`"
```

Comment:
Thanks for the test pointers, I've added some.
The problem message would be misleading as the domain needs to be updated as well.